src/test/java/api/loans/DueDateScheduledNoticesTests.java

```diff
@@ -621,4 +631,138 @@ public void scheduledOverdueNoticesShouldBeDeletedAfterOverdueFineIsCharged() {
       .atMost(1, TimeUnit.SECONDS)
       .until(scheduledNoticesClient::getAll, hasSize(0));
   }
+
+  @Test
+  public void scheduledOverdueNoticesShouldBeDeletedAfterAgedToLost() {
+    NoticePolicyBuilder noticePolicy = createSendNoticePolicy();
+    use(noticePolicy);
+    createScheduledNotice(new DateTime(2020, 1, 1, 12, 0, 0, UTC));
+
+    var agedToLostLoan = ageToLostFixture.createAgedToLostLoan(noticePolicy);
+    var dueDate = getDateTimeProperty(agedToLostLoan.getLoan().getJson(), "dueDate");
+
+    scheduledNoticeProcessingClient.runLoanNoticesProcessing(dueDate.plusDays(1));
+
+    Awaitility.await()
+      .atMost(1, TimeUnit.SECONDS)
+      .until(scheduledNoticesClient::getAll, empty());
+  }
+
+  @Test
+  public void scheduledOverDueNoticeShouldBeDeletedAfterClaimedReturned() {
+    NoticePolicyBuilder noticePolicy = createSendNoticePolicy();
+    use(noticePolicy);
+    IndividualResource loan = createScheduledNotice(now());
+
+    claimItemReturnedFixture
+      .claimItemReturned(new ClaimItemReturnedRequestBuilder()
+        .forLoan(loan.getId().toString())
+        .withItemClaimedReturnedDate(now()));
+
+    var dueDate = getDateTimeProperty(loan.getJson(), "dueDate");
+    scheduledNoticeProcessingClient.runLoanNoticesProcessing(dueDate.plusDays(1));
+
+    Awaitility.await()
+      .atMost(1, TimeUnit.SECONDS)
+      .until(scheduledNoticesClient::getAll, empty());
+  }
+
+  @Test
+  public void scheduledOverdueNoticesShouldBeDeletedAfterDeclaredLost() {
+    NoticePolicyBuilder noticePolicy = createSendNoticePolicy();
+    use(noticePolicy);
+    IndividualResource loan = createScheduledNotice(now());
+
+    declareLostFixtures.declareItemLost(loan.getJson());
+    var dueDate = getDateTimeProperty(loan.getJson(), "dueDate");
+    scheduledNoticeProcessingClient.runLoanNoticesProcessing(dueDate.plusDays(1));
```

Comment:
Try setting smaller delay, like 1 second

Reply:
1 hour is possible as min

Reply:
see method `DateTime#plusSeconds`